generated/nirfmxinstr/nirfmxinstr.proto

```diff
@@ -74,7 +74,6 @@ service NiRFmxInstr {
   rpc GetSelfCalibrateLastTemperature(GetSelfCalibrateLastTemperatureRequest) returns (GetSelfCalibrateLastTemperatureResponse);
   rpc GetSignalConfigurationNames(GetSignalConfigurationNamesRequest) returns (GetSignalConfigurationNamesResponse);
   rpc Initialize(InitializeRequest) returns (InitializeResponse);
-  rpc InitializeWithChannel(InitializeWithChannelRequest) returns (InitializeWithChannelResponse);
```

Comment:
Please provide a better description because we need to include it in the release notes. My understanding is that you think this is OK to remove because:

1. It was added very recently (2 months ago).
2. It is also being deleted from the C API (so there's nothing for grpc-device call and you've decided to break at the driver API level).
3. Even though it is binary breaking, the failure mode is clear and clean (failure to initialize).

I don't see a justification for *why* the method is being removed that I understand. You linked a task that said "Channel information from separate parameter is not expected now", which doesn't explain it to me. Please come up with a message that you would share with customers explaining why this API is removed.

Reply:
update the description